Containerfile

```diff
@@ -1,17 +1,46 @@
-FROM registry.access.redhat.com/ubi9/ubi-minimal AS build
+FROM registry.fedoraproject.org/fedora:latest AS build_python
 
-RUN \
-    microdnf install -y \
+RUN dnf install -y \
         python3-pip \
         python3-pyyaml \
-        && microdnf clean all
+        && dnf clean all
 
 WORKDIR /src
 
 COPY pyproject.toml /src/
 COPY src /src/src
 
 RUN pip install --no-dependencies .
-RUN rm -rf src/*
+
+# ----
+FROM registry.access.redhat.com/ubi9/go-toolset:latest AS build_go
+
+ARG CONTAINERS_STORAGE_THIN_TAGS="containers_image_openpgp exclude_graphdriver_btrfs exclude_graphdriver_devicemapper"
```

Comment:
Can we somehow avoid duplicating this? It's already in the Makefile iirc?

Reply:
I wanted to do that but I'm not sure how to do that. For me it's better to have a sane fallback if someone builds this `Containerfile` directly.
having some grep here and there or accepting that `Containerfile` can't be built without that knowledge seems both bad to me

Reply:
grepping from the Containerfile into the Makefile could work - if that's nicer? 🤷‍♂️

Reply:
Sorry, maybe I'm just confused by we call `make` in the Containerfile, why is it not picking up the options that we set in the makefile, i.e. why do we need them here again? And sorry for being to nitpicky about this but this list irks me a great deal already, we have it in bootc-image-builder too and now we have it here in two places, I would really like to minimize the places we need to update :)

Reply:
🫣 to no surprise, you are right!
This was due to refactoring (make was not called before) and totally not needed… Thanks!